### ISSUE TITLE
Using MASt3R instead of DUSt3R

### ISSUE THREAD

**Issue by anekha** [GaussianObject/GaussianObject#61]
Hello
I wanted to confirm if we need to amend any files/code to use MASt3R instead of DUSt3R.
I see that in the file pred_poses_mast3r.py the poses are saved as the below:

'    np.save(os.path.join(scene_path, f'dust3r_depth_{sparse_num}.npy'), depths)
    confidences = np.array([confidence.detach().cpu().numpy() for confidence in confidences])
    np.save(os.path.join(scene_path, f'dust3r_confidence_{sparse_num}.npy'), confidences)'

as per screenshot attached. Additionally, all the training files reference DUSt3R. So I wanted to check whether I would just run the MASt3R poses and they are saved over the DUSt3R and I run everything as DUSt3R?

That is, I run the below:

!python train_gs.py -s data/jewelry/ring_two \
    -m output/gs_init/ring_two \
    -r 1 --sparse_view_num 12 --sh_degree 9 \
    --init_pcd_name dust3r_12 \
    --white_background --use_dust3r
<img width="490" alt="Screenshot 2024-11-04 at 11 55 30" src="https://github.com/user-attachments/assets/d6b28350-f10a-4713-a12e-a4a7601ff2a6">

<img width="862" alt="Screenshot 2024-11-04 at 11 53 39" src="https://github.com/user-attachments/assets/e2695139-c319-4dfd-8049-a68682ef0bbf">

Thanks

**Comment by UranusITS** [comment]
Thank you for your question! You can proceed as described without any issues. MASt3R is fully compatible with the existing pipeline, and the poses and point clouds generated by MASt3R are more stable compared to DUSt3R.

**Comment by 7rwang** [comment]
The command ` -- use_dust3r` keeps unchanged, even if we're using mast3r?

**Comment by 7rwang** [comment]
> The command ` -- use_dust3r` keeps unchanged, even if we're using mast3r?

Ok never mind. I have understood
